app/viewmodels/models/ListViewModel.scala

```diff
@@ -42,33 +45,55 @@ object ListRow {
     text,
     change = Some(ListRowLink(changeUrl, changeHiddenText)),
     remove = Some(ListRowLink(removeUrl, removeHiddenText)),
-    view = None
+    view = None,
+    check = None
   )
 
   def view(text: DisplayMessage, url: String, hiddenText: Message): ListRow =
     ListRow(
       text,
       change = None,
       remove = None,
-      view = Some(ListRowLink(url, hiddenText))
+      view = Some(ListRowLink(url, hiddenText)),
+      check = None
+    )
+
+  def check(text: DisplayMessage, url: String, hiddenText: Message): ListRow =
+    ListRow(
+      text,
+      change = None,
+      remove = None,
+      view = None,
+      check = Some(ListRowLink(url, hiddenText))
     )
 
   def viewNoLink(text: DisplayMessage, value: DisplayMessage): ListRow =
     ListRow(
       text,
       change = None,
       remove = None,
-      view = Some(ListRowNoLink(value))
+      view = Some(ListRowNoLink(value)),
+      check = None
     )
 }
 
 case class ListViewModel(
   inset: DisplayMessage,
-  rows: List[ListRow],
+  sections: List[Section],
```

Comment:
main change - list pages now have a list of sections instead of rows, each section is just a heading and a list of rows. This can be expanded to house section specific elements like pagination if needed later on